demos/026.ldio-workbench-upgrade/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     container_name: ${USECASE_NAME:-upgrade-ldio-workbench}_test-message-generator
     image: ghcr.io/informatievlaanderen/test-message-generator:${TEST_MESSAGE_GENERATOR_TAG:-latest}
     environment:
-      - TARGETURL=
+      - TARGETURL=http://old-ldio:8080/pipeline
```

Comment:
please revert this .. both manual and auto test require you to write a `./data/TARGETURL` file instead